src/Components/FunFact.js

```diff
@@ -0,0 +1,113 @@
+import app from '../firebase';
```

Comment:
Consider adding comments above the styling, modules, config, and components imports, similar to how you did it in your App.js file.

Reply:
Great point!